microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/definition/AbstractAnnotationDefinitionParticipant.java

```diff
@@ -47,18 +48,35 @@
 
 	private final String annotationName;
 
-	private final String annotationMemberName;
+	private final String[] annotationMemberNames;
+
+	protected Function<String, String> propertyReplacer;
 
 	/**
 	 * The definition participant constructor.
-	 * 
-	 * @param annotationName       the annotation name (ex :
-	 *                             org.eclipse.microprofile.config.inject.ConfigProperty)
-	 * @param annotationMemberName the annotation member name (ex : name)
+	 *
+	 * @param annotationName        the annotation name (ex :
+	 *                              org.eclipse.microprofile.config.inject.ConfigProperty)
+	 * @param annotationMemberNames the supported annotation member names (ex :
+	 *                              name)
 	 */
-	public AbstractAnnotationDefinitionParticipant(String annotationName, String annotationMemberName) {
+	public AbstractAnnotationDefinitionParticipant(String annotationName, String[] annotationMemberNames) {
 		this.annotationName = annotationName;

```

Comment:
`this(annotationName, annotationMemberNames, PropertyReplacerStrategy.NULL_REPLACER);`